python/ray/util/sgd/v2/migration/sgd-migration-guide.md

```diff
@@ -0,0 +1,224 @@
+# Ray SGD: v1 to v2 migration guide
+
+In Ray 1.7, we are rolling out a new and more streamlined version of Ray SGD. Ray SGD v2 focuses on usability and composability- it has a much simpler API, has support for more deep learning backends, integrates better with other libraries in the Ray ecosystem, and will continue to be actively developed with more features. 
+
+This guide will cover to easily migrate existing code from Ray SGD v1 to Ray SGD v2. If you are new to Ray SGD as a whole, you should get started with Ray SGD v2 directly: https://docs.ray.io/en/master/raysgd/v2/raysgd.html. 
+
+For a full list of features that Ray SGD v2 provides, please check out the user guide: https://docs.ray.io/en/master/raysgd/v2/user_guide.html. 
+
+If there are any issues or anything missing with this guide or any feedback on Ray SGD v2 overall, please file a Github issue on the Ray repo: https://github.com/ray-project/ray/issues!
+
+## What are the API differences?
```

Comment:
Change this section to `Overview`? The transition from this section to the following section wasn't so clear and I think the reason why was that this section doesn't include concrete migration steps while the following sections do.

Reply:
I added the small snippet at the end saying what the next sections are going to be so that should cover the confusion I believe.